src/theme.rs

```diff
@@ -266,6 +266,10 @@ pub(crate) struct IntroSlideStyle {
     /// The style of the author line.
     #[serde(default)]
     pub(crate) author: AuthorStyle,
+
+    /// Whether we want a footer in the intro slide.
+    #[serde(default)]
+    pub(crate) footer: Option<bool>,
```

Comment:
I was going to suggest renaming this to `footer_on_intro` or something like that, to make it more obvious that we are not skipping the footer always, just on the intro slide, but I've seen that this will be parsed from a YAML section that is already called `intro_slide` so it's clear that the setting only applies to that slide. No comments then! xD